iOS/MSMapsModules/MSMapsModules/MSMapGeoJsonParser/MSMapGeoJsonLayer.m

```diff
@@ -46,4 +52,88 @@ - (void)setStrokeColor:(UIColor *)newStrokeColor {
 	}
 }
 
+- (void)setStrokeDashed:(BOOL)newIsStrokeDashed {
+	if (_strokeDashed != newIsStrokeDashed) {
+		_strokeDashed = newIsStrokeDashed;
```

Comment:
This doesn't look properly formatted. Merge the other change to master then merge master to this branch and run format again.